packages/@react-aria/label/docs/useField.mdx

```diff
@@ -0,0 +1,77 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/label';
+import {FunctionAPI, HeaderInfo} from '@react-spectrum/docs';
+import packageData from '@react-aria/label/package.json';
+
+```tsx import
+import {useField} from '@react-aria/label';
+```
+
+---
+category: Utilities
+keywords: [label, form, description, error, field, aria]
+after_version: 3.1.3
+---
+
+# useField
+
+<p>{docs.exports.useField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useField']} />
+
+## API
+
+<FunctionAPI function={docs.exports.useField} links={docs.links} />
+
+## Example
+
+The `useField` hook associates a form control with a label, and an optional description and/or error message. This is useful for providing context about how users should fill in a field, or a validation message. `useField` takes care of creating ids for each element and associating them with the correct ARIA attributes (`aria-labelledby` and `aria-describedby`).
+
+**Note**: Many other React Aria hooks such as [useTextField](useTextField.html), [useSelect](useSelect.html), and [useComboBox](useComboBox.html) already include support for description and error message elements. If you're using one of those hooks, there's no need to use `useField`.
```

Comment:
I'm prefer we didn't say "Note", but I realize if we don't do this and more problems will happen.